python/lsst/meas/modelfit/display/cModelDisplay.py

```diff
@@ -0,0 +1,264 @@
+    fig.subplots_adjust(left=0.25, right=0.8)
+    lCBar = fig.add_axes([0.1, 0.15, 0.05, 0.7])
+    rCBar = fig.add_axes([0.85, 0.15, 0.05, 0.7])
+
+    # Populate just the exposures in the appropriate places
+    for i in range(3):
+        axes[i, 0].imshow(subImage.getArray(), vmin=subImMin, vmax=subImMax)
+
+    # Populate dev panels
+    axes[0, 1].imshow(devIm.getArray(), vmin=subImMin, vmax=subImMax)
+    axes[0, 2].imshow(devResidual, vmin=resImMin, vmax=resImMax, cmap="BrBG")
+
+    # Populate exp panels
+    axes[1, 1].imshow(expIm.getArray(), vmin=subImMin, vmax=subImMax)
+    axes[1, 2].imshow(expResidual, vmin=resImMin, vmax=resImMax, cmap="BrBG")
+
+    # Populate joint pannels
+    axes[2, 1].imshow(jointIm.getArray(), vmin=subImMin, vmax=subImMax)
+    axes[2, 2].imshow(jointResidual, vmin=resImMin, vmax=resImMax, cmap="BrBG")
+
+    axes[0, 0].set_title("Image")
+    axes[0, 1].set_title("Model")
+    axes[0, 2].set_title('Residuals')
+
+    axes[0, 0].set_ylabel("Dev")
+    axes[1, 0].set_ylabel("Exp")
+    axes[2, 0].set_ylabel("Joint")
+
+    fig.colorbar(axes[0, 0].get_images()[0], lCBar)
+    lCBar.yaxis.set_ticks_position('left')
+    fig.colorbar(axes[maxDifferenceIndex, 2].get_images()[0], rCBar)
+
+    plt.show()
+
+
+def buildCModelImages(exposure, record, cModelConfig):
+    """ Create Images out of the CModel for the given record
+
+    Parameters
+    ----------
+    exposure : `lsst.afw.image.Exposure`
+        Exposure object that contains the source which was  modeled
+    record : `lsst.afw.table.SourceRecord`
+        Record object which contains the measurements made on the source
+    cModelConfig : `lsst.meas.modelfit.CModel(SingleFrame/Forced)Config`
+        Configuration object of the CModel plugin used in the measurement
+        process
+
+    Returns
+    -------
+    subImage : `lsst.afw.image.ImageD`
+        Sub image of the original data taken from a region defined by the
+        bounding box of the footprint for the object defined in the given
+        source record
+    devIm : `lsst.afw.image.ImageD`
+        Image created from the dev component of the CModel for the supplied
+        record at the same pixel locations as subImage
+    expIm: `lsst.afw.image.ImageD`
+        Image created from the exp component of the CModel for the supplied
+        record at the same pixel locations as subImage
+    jointIm :
+        Image created from the joint fit of the dev and exp components of the
+        CModel for the supplied record at the same pixel locations as subImage
+    """
+
+    dev, exp, jointDev, jointExp, refFlux = reconstructCModel(exposure, record,
+                                                              cModelConfig)
+    # Get exposure cutout
+    footBBox = record.getFootprint().getBBox()
+    subImage = afwImage.ImageD(afwImage.ImageD(exposure.getImage(), True),
+                               footBBox)
```

Comment:
Should probably use `ImageF` instead; that's what we always fit to, and it's what the original image is.  That should remove the need to call the constructor twice, too.

Strongly recommend using kwargs for any image constructor arguments that aren't passed as named variables for clarity (i.e. `True` in this case).